src/main/java/com/example/geekshubs/VoidFeature.java

```diff
@@ -10,5 +12,6 @@ public void nothingToDo() {
     System.out.println("Nothing to do here");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)